server/http_integration_test.go

```diff
@@ -195,6 +196,14 @@ func cleanup(shutdown func(), helper *test.Helper) {
 	}
 }
 
+func nilToString(in string) string {
```

Comment:
Seems to be obsolete. You could pass the result value to `Errorf("state: %q")` to get a quoted one: `state: ""` to show up an unexpected empty string.